src/BinaryParsers/PEBinary/PortableExecutable/PE.cs

```diff
@@ -789,5 +789,66 @@ public bool IsWixBinary
                 return this.isWixBinary.Value;
             }
         }
+
+        public bool IsChecksumAlgorithmSecureForPortablePdb()
+        {
+            const string sha256 = "8829d00f-11b8-4213-878b-770e8597ac16";
+            var sha256guid = new Guid(sha256);
+
+            if (this.peReader.TryOpenAssociatedPortablePdb(

```

Comment:
>TryOpenAssociatedPortablePdb [](start = 30, length = 28)

eventually pdb location logic needs to understand binskim's '--local-symbol-directories' command-line arguments. let's open an issue and take this in a future change.

Reply:
issue opened: [https://github.com/microsoft/binskim/issues/338](https://github.com/microsoft/binskim/issues/338)

---
In reply to: [563213769](https://github.com/microsoft/binskim/pull/335#discussion_r563213769) [](ancestors = 563213769)